docs/intro.txt

```diff
@@ -11,9 +11,14 @@ and Windows. To install, run:
 
     pip install pytket-quantinuum
 
+
+.. note:: pytket-quantinuum is not intended to be used with Quantinuum Nexus. For guidance on how to access H-Series through Nexus, please see the `Nexus documentation <https://docs.quantinuum.com/nexus>`_ and the `qnexus <https://pypi.org/project/qnexus/>`_ Python package.
```

Comment:
Just wanted to clarify my reasoning for my original wording. Copied below

> pytket-quantinuum is not compatible with Quantinuum Nexus. To use your Nexus account please see the Nexus documentation <https://docs.quantinuum.com/nexus>_ and the qnexus <https://pypi.org/project/qnexus/>_ Python package.

"pytket-quantinuum is not intended to be used with Quantinuum Nexus" - to me this possibly implies that pytket-quantinuum can be used with Nexus but we advise against it. This isn't the case - pytket-quantinuum flat out doesn't work with Nexus.

To be clear, the main reason I'm requesting this change is to catch users that have been given access to Nexus, and end up on this page by mistake thinking that its a client for Nexus. Therefore they might not even be looking to target H-series.

@irfankhan10 do you have any thoughts? Am I being too pedantic?

Reply:
For context: I'm working with Irfan to reduce friction between H-series and Nexus docs, this seems relevant.

Reply:
Feel free to dismiss my pedantry if desired - will respect whatever decision TKET team makes! Just a bit wary of users getting lost in our documentation

Reply:
Makes sense, I'll go with incompatible as you suggest.